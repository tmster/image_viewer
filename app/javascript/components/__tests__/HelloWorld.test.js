import React from 'react';
import { shallow } from 'enzyme';

import HelloWorld from '../HelloWorld';

it('renders name', () => {
  const node = shallow(<HelloWorld greeting="example" />);
  expect(node).toMatchSnapshot();
});
