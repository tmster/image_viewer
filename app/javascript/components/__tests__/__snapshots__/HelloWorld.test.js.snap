// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders name 1`] = `ShallowWrapper {}`;
